README.md

```diff
@@ -77,39 +77,65 @@ X | ExoComp
 Z | Daily Missions
 ` | Open Alliance Chat - Side of Screen
 
-## Building
+## Contributing / Building
 
-First clone and initialise the repository:
+If you wish to contribute to the project, or simply compile the DLL yourself, please see [CONTRIBUTING.md](CONTRIBUTING.md)
 
-```bash
-git clone https://github.com/tashcan/bob.git
-cd bob
-git submodules update --recursive --init
-```
+## Installation / Runtme
 
-This will download three submodules and further submodules that they have, so this will take some time to complete.
+Both the DLL and the toml file must be placed in the STFC game folder which is always located
+at `C:\Games\Star Trek Fleet Command\Star Trek Fleet Command\default\game` as the following
+names:
 
-If you have Visual Studio Code, open the folder and it should ask you to install the CMake extensions bundle.  Once the extensions are installed, you can build the project by pressing `F7` or right clicking on the `CMakeLists.txt` file and selecting `Build All Projects`.  This may ask you to configure the project and pick various items to use in the build.  That should only occur once.
+- community_patch_settings.toml
+- version.dll
 
-If you do not have Visual Studio Code, this project uses CMake, so the simplest way to build it on Windows:
+The pre-compiled DLL can be downloaded from the official assets hosted at:
```

Comment:
As I mentioned in the other PR. I would probably just point to the Releases here, as that'll be where things will be for updates going forward.

Reply:
I have been slow for the past day or two... I wondered where "here" was... thinking discord... until I just clicked... I'll have this updated shortly.